integration-guides/webfleet.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Bridgestone Webfleet
+---
+
+To connect your WebFleet telematics device integration with PathStack, you will need to be provided with your Account name, a Username, password and API Key for accessing the API. 
```

Comment:
Capitals should be consistent, so in this case Account Name is capitalised, as it Password